File: src/main/java/jp/artan/japanesefoodmod/init/tabs/FruitTreeCreativeTab.java
package jp.artan.japanesefoodmod.init.tabs;

import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.ItemStack;

import static jp.artan.japanesefoodmod.init.BlockInit.BANANA;

public class FruitTreeCreativeTab extends CreativeTabs {

    public FruitTreeCreativeTab() {
        super("japanesefoodfruittree");
    }

    @Override
    public ItemStack getTabIconItem() {
        return new ItemStack(BANANA.log);
    }

}

File: src/main/java/jp/artan/japanesefoodmod/init/tabs/FruitTreeCreativeTab.kt
package jp.artan.japanesefoodmod.init.tabs

import jp.artan.japanesefoodmod.init.BlockInit
import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.ItemStack

class FruitTreeCreativeTab : CreativeTabs("japanesefoodfruittree") {
    override fun getTabIconItem(): ItemStack {
        return ItemStack(BlockInit.BANANA.log)
    }

}
